tasks/main.yml

```diff
@@ -133,7 +133,7 @@
     --filename={{ osbs_openshift_home }}/{{ inventory_hostname }}-{{ osbs_namespace }}-rolebinding-{{ item.item.name }}.yml
   environment: "{{ osbs_environment }}"
   with_items: "{{ yaml_rolebindings.results }}"
-  when: item.changed
+  when: yaml_rolebindings.changed and item.changed
```

Comment:
I don't understand. If none of the items changed, why would `yaml_rolebindings.changed` be true?

Reply:
If none of the items changed `yaml_rolebindings` won't have `results`, but would have `changed: false` - and that would cause the error (at least it did for pruner on OSD flow, so I copied the fix there - that worked for orchestrator and worker)

Reply:
Interesting. Thanks for clarifying.